connector_easypost/__openerp__.py

```diff
@@ -11,8 +11,11 @@
     'license': 'AGPL-3',
     'website': 'https://laslabs.com',
     'depends': [
+        'stock',
         'connector',
-        'stock_delivery_label_new',
+        'stock_picking_rate',
+        'stock_picking_package_info',
+        'base_delivery_carrier_label',
```

Comment:
The issue is that you need to define the 9.0 branch explicitly. 10.0 is now default.